setup.py

```diff
@@ -20,6 +20,6 @@
         'doodle.shared'],
       include_package_data=True,
       install_requires = [
-          'numpy','matplotlib','pandas==1.5.3','pydicom','openpyxl','rt_utils','scikit-image', 'simpleitk'
+          'numpy','matplotlib','pandas','pydicom','openpyxl','rt-utils','scikit-image', 'simpleitk'
```

Comment:
Any reason why we are more flexible with pandas version? I just want to confirm this is intended.